api/swagger.yml

```
@@ -1938,6 +1938,36 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PagedData"
+        500:
+          $ref: "#/components/responses/Internal"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        403:
+          $ref: "#/components/responses/Forbidden"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)